includes/definitions/drac.yaml

```diff
@@ -2,11 +2,11 @@ os: drac
 text: 'Dell DRAC'
 icon: dell
 type: appliance
-mib_dir: dell
+mib_dir: drac
```

Comment:
Mibs are ordered by vendor so dell is the correct value

Reply:
in that case we need to delete the MIBs in the folder drac and copy the two files to the dell folder. Does this make sense as DELL has a lot of different MIBs. I don't know if they may interfere?

Reply:
I just pushed an update with the correct folders